src/app/lib/config/services/thai.js

```diff
@@ -6,6 +6,7 @@ import '@bbc/moment-timezone-include/tz/Asia/Bangkok';
 
 const service = {
   lang: `th`,
+  isoLang: `th`,
```

Comment:
I think you can just use lang as it's identical. We shouldn't add an attribute for only one Service.

Reply:
I can see that the `<link rel="alternative" hreflang='..'/>` is only applied for services with the `isoLang` attribute present. What is the desired output? Should `<link rel="alternative ... />` be present for all services.

Reply:
Yes, I believe it should be present for all services except Pidgin, I presume though that this implementation choice in config is the way to exclude Pidgin. So you've not introduced it, I retract my concern as you're just repeating what we already do.

Reply:
I deleted the `isoLang` keys from services except `pidgin`. It looks like the `isoLang` will be the same as `lang` for the other services. We can do a null check on `isoLang` and have `<link rel="alternative ... />` present for all services except pidgin.

Reply:
I understand the previous way of using `isoLang` was confusing but IMO this is worse.
With this change (if I'm reading it right), `isoLang` is only used to suppress `<link rel="alternative" hreflang='..'/>`, which you do by setting it to `null`?

If we're going to use this PR to refactor this part of config, can we look at finding a way which is more descriptive, not less? Or can we just focus on the issue at hand (fixing Thai metadata on live) and deal with our tech debt separately?

Reply:
Am happy to revert and let this PR focus on updating the `thai` metadata. Can you open an issue to address the tech dept.

Reply:
Thanks Isaac :) raised https://github.com/bbc/simorgh/issues/3405